PowerEditor/installer/nativeLang/english.xml

```diff
@@ -547,41 +547,41 @@ The comments are here for explanation, it's not necessary to translate them.
-                    <Item id="43040" name="Next mark using 2nd style"/>
-                    <Item id="43041" name="Next mark using 3rd style"/>
-                    <Item id="43042" name="Next mark using 4th style"/>
-                    <Item id="43043" name="Next mark using 5th style"/>
-                    <Item id="43044" name="Next mark created with Mark"/>
-                    <Item id="43055" name="Copy Styled Text - 1st Style"/>
-                    <Item id="43056" name="Copy Styled Text - 2nd Style"/>
-                    <Item id="43057" name="Copy Styled Text - 3rd Style"/>
-                    <Item id="43058" name="Copy Styled Text - 4th Style"/>
-                    <Item id="43059" name="Copy Styled Text - 5th Style"/>
-                    <Item id="43060" name="Copy Styled Text - All Styles"/>
-                    <Item id="43061" name="Copy Styled Text - Find Style (Marked)"/>
+                    <Item id="43022" name="Style all using 1st style"/>
+                    <Item id="43024" name="Style all using 2nd style"/>
+                    <Item id="43026" name="Style all using 3rd style"/>
+                    <Item id="43028" name="Style all using 4th style"/>
+                    <Item id="43030" name="Style all using 5th style"/>
+                    <Item id="43062" name="Style one using 1st style"/>
+                    <Item id="43063" name="Style one using 2nd style"/>
+                    <Item id="43064" name="Style one using 3rd style"/>
+                    <Item id="43065" name="Style one using 4th style"/>
+                    <Item id="43066" name="Style one using 5th style"/>
+                    <Item id="43023" name="Clear 1st style"/>
+                    <Item id="43025" name="Clear 2nd style"/>
+                    <Item id="43027" name="Clear 3rd style"/>
+                    <Item id="43029" name="Clear 4th style"/>
+                    <Item id="43031" name="Clear 5th style"/>
+                    <Item id="43032" name="Clear all styles"/>
+                    <Item id="43033" name="Previous style of 1st style"/>
+                    <Item id="43034" name="Previous style of 2nd style"/>
+                    <Item id="43035" name="Previous style of 3rd style"/>
+                    <Item id="43036" name="Previous style of 4th style"/>
+                    <Item id="43037" name="Previous style of 5th style"/>
+                    <Item id="43038" name="Previous style of Mark (Find dialog)"/>
+                    <Item id="43039" name="Next style of 1st style"/>
+                    <Item id="43040" name="Next style of 2nd style"/>
+                    <Item id="43041" name="Next style of 3rd style"/>
+                    <Item id="43042" name="Next style of 4th style"/>
+                    <Item id="43043" name="Next style of 5th style"/>
+                    <Item id="43044" name="Next style of Mark (Find dialog)"/>
+                    <Item id="43055" name="Copy Styled Text of 1st Style"/>
```

Comment:
In all copy this "Style" can be given with lowercase, in accordance with all other cases.